lms/lib/comment_client/user.py

```diff
@@ -115,25 +115,33 @@ def active_threads(self, query_params={}):
         return response.get('collection', []), response.get('page', 1), response.get('num_pages', 1)
 
     def subscribed_threads(self, query_params={}):
+        response = self._perform_subscribed_threads_request(query_params)
+
+        return utils.CommentClientPaginatedResult(
+            collection=response.get('collection', []),
+            page=response.get('page', 1),
+            num_pages=response.get('num_pages', 1),
+            thread_count=response.get('thread_count', 0)
+        )
+
+    def is_subscribed_to_thread(self, thread_id):
+        thread_ids = [thread['id'] for thread in self._perform_subscribed_threads_request()['collection']]
+        return thread_id in thread_ids
+
+    def _perform_subscribed_threads_request(self, query_params={}):
```

Comment:
For this method, what happens if the user is subscribed to 300 threads and this one doesn't come as part of the first page?

Reply:
good question, I remember thinking about that, and then I never followed up.  I'll check!

Reply:
Thanks, Sandy, it's addressed now.